tests/bson/bson-toPHP-017.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+MongoDB\BSON\toPHP(): int64 values are returned as int scalars
```

Comment:
I'll note that 64-bit values on a 32-bit platform are an edge case where Int64 objects would still be returned, but I don't think we need to demonstrate that in a test. If you want you can add a note above `'int64' => new MongoDB\BSON\Int64(123),` explaining that you're using a 32-bit value to simplify testing.